api/v1beta3/auth_config_types.go

```diff
@@ -612,6 +612,7 @@ type KubernetesSubjectAccessReviewAuthorizationSpec struct {
 	User *ValueOrSelector `json:"user,omitempty"`
 
 	// Groups the user must be a member of or, if `user` is omitted, the groups to check for authorization in the Kubernetes RBAC.
+	// Deprecated: Use authorizationGroups instead.
```

Comment:
Can you please `make manifests` again, so we catch this in the OAS too?

Reply:
Sorry about that. Fixed it by running `make manifests`. I also added a check for uncommitted changes in go-test.yaml action.